### PR TITLE
Update carbonapi version

### PR DESCRIPTION
**Pull request by Tetrergeru**
Updated carbonapi version due to panics caused by older on some invalid targets.

Added error unwrapping, because errors are wrapped in newer versions of carbonapi. 

### Can not be merged

Due to another bug in carbonapi that makes checker panic on existing valid triggers.

### CODE REVIEW

File: test_res.txt [hunk 1]
```diff
@@ -0,0 +1,528 @@
+ok  	github.com/moira-alert/moira	(cached)
```
Comment (kissken) [inline test_res.txt:1]: hmmm, added into gitignore?

Comment (Tetrergeru) [reply test_res.txt]: I acidently added this file to git. Deleted :ok_hand:

**Comment by Dimedrolity** [comment]
>Updated carbonapi version due to panics caused by older on some invalid targets.

@Tetrergeru What the reason is mean? Please explain where the panics were on old version of carbonapi

File: metric_source/local/errors.go [hunk 2]
```diff
@@ -33,7 +34,7 @@ func (err ErrUnknownFunction) Error() string {
 
 // isErrUnknownFunction checks error for carbonapi.errUnknownFunction
 func isErrUnknownFunction(err error) bool {
-	switch err.(type) {
+	switch merry.Unwrap(err).(type) {
```
Comment (kissken) [inline metric_source/local/errors.go:37]: missed any tests on this case?

let's add?

Comment (Tetrergeru) [reply metric_source/local/errors.go:37]: This change was caused by package update. Without it trigger tests start to fail

**Comment by Tetrergeru** [comment]
@Dimedrolity, old version of carbonapi successfully parses invalid target and then panics on Metrics method call

**Comment by Dimedrolity** [comment]
@Tetrergeru Ok, thanks. Provide example of invalid target, please

**Comment by Tetrergeru** [comment]
@Dimedrolity, the bug was discovered with target `aliasByNode(movingMin(Infrastructure.ClusterConfig.cloud.ClusterConfig_SloMeter.nonEquivalent.V2.*), '10min'), 6)`

**Comment by Dimedrolity** [comment]
@Tetrergeru Ok, I suggest to add test to that target to ensure that bug was fixed. It seems need to test _metric_source/local/local.go_ Fetch method, because only there is the call to Metrics method

**Comment by kissken** [comment]
@Dimedrolity panic is not primary reason for update carbon-api, it is secondary and background. And PR have necessary title :)

**Comment by Dimedrolity** [comment]
@kissken I don't see another reason in title or description of PR. So what is the primary reason for update carbon-api?

**Comment by kissken** [comment]
@Dimedrolity technical debt

**Comment by Dimedrolity** [comment]
@kissken Ok, thanks.

Do you think [this is overhead](https://github.com/moira-alert/moira/pull/800#issuecomment-1309767672)?

**Comment by kissken** [comment]
@Dimedrolity aha :)

**Review by Dimedrolity — APPROVED** [review]
Well

**Comment by kissken** [comment]
buuuut @Tetrergeru let's mark as draft, end explain, why we cannot merge, please

**Comment by Tetrergeru** [comment]
@kissken, Done

**Comment by Dimedrolity** [comment]
> Due to another bug in carbonapi that makes checker panic on existing valid triggers.

Oh😒
How many existing valid triggers are panic? Why they panic?

**Comment by kissken** [comment]
@Tetrergeru heeeey, let's close this PR? %)

**Pull request closed without merging**